cloud/endagaweb/views/dashboard.py

```diff
@@ -192,7 +208,8 @@ def billing_view(request):
     msgs = messages.get_messages(request)
     for m in msgs:
         if "billing_resp_code" in m.tags:
-            context[m.message] = True # pass the message on to the template as-is
+            context[
+                m.message] = True  # pass the message on to the template as-is
```

Comment:
this is ugly formatting, you can revert the change (and put the comment onto the preceding line)